### ISSUE TITLE
A11y_Visual Studio Code Client_Sep2021_Debug Adapter Protocol_HomePage_HighContrast: In Footer section "Microsoft" link is not visible in high contrast black .

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/debug-adapter-protocol#208]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag**:#A11yTCS;#DesktopWeb;#Win10;#WCAG1.4.3;#FTP;#A11yMAS;#ChromiumEdge;#HighContrast;#A11ySev1;#P2_Visual Studio Code Clien_Sep2021;#Visual Studio Code Client;#P2_WCP;#AILimited;

## **Environment Details:**

Application Name: Visual Studio Code Client
#URL: https://microsoft.github.io/debug-adapter-protocol
Browser Versions: Microsoft Edge Version 93.0.961.38 (Official build) (64-bit)
Windows Version: Windows10

**Example Page URL**: https://microsoft.github.io/debug-adapter-protocol

## **Repro Steps:**
1.	Open the URL: https://microsoft.github.io/debug-adapter-protocol
2.	Debug Adapter Protocol home screen will get open.
3.	Apply high contrast black and tab to footer section and verify the issue.

## **Actual:**

In Footer section "Microsoft" link not visible in high contrast black .
Focus outline is also not visible on  links: "*star" link, "637" link, "Watch" link at footer section and "*star" link, "637" link at header section.

**Same issue is also observed throughout the application where same links and controls present**.

## **Expected:**

All the controls should be visible on high contrast black mode.

## **User Impact:**

User are able to see controls on high contrast mode.

## **Attachment:**
![#208](https://user-images.githubusercontent.com/60422426/133750907-1fa990c3-2362-4ed9-96c7-3125954d6ea4.gif)

**Comment by sana-ajani** [comment]
cc @gregvanl can you triage this one? I don't have a similar dark high contrast mode on my mac to be able to repro.

**Comment by rzhao271** [comment]
Issue confirmed.

TODO: 
- [ ] Swap out the svg with the two pngs from the vscode website
- [ ] Use a `picture` tag to use the dark mode png when dark mode is preferred

**Comment by NiharikaTipirishetty** [comment]
#P2_debugadapterprotocol_Web_Sep2021;

**Comment by prabhudevu** [comment]
I have verified the issue in the environment: https://microsoft.github.io/debug-adapter-protocol
issue 1:"Microsoft link" in the footer issue is resolved, But the issue 2: " Focus outline is not visible on links: "*star" link, "637" link, "Watch" link at footer section and "*star" link, "637" link at header section." this issue is not resolved. you can observe that this issue is also mentioned in the "Actual Section". Added attachment for the reference. This issue is also observed with Desert theme. @connor4312 Requesting you to open the bug again as this bug is not completely resolved



![Screenshot (337)](https://user-images.githubusercontent.com/98821792/178501836-1eeed2b0-2052-4a39-bec0-ad4f134af780.png)


![HCgif](https://user-images.githubusercontent.com/98821792/178500343-44f38fda-5ca0-4f66-a076-bf1efc84bd2d.gif)

**Comment by rzhao271** [comment]
Will keep this issue closed and track the star link bug at #206
